### ISSUE TITLE
Add following comment to scaphocephaly

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#846]
Summarize the following comments, this will help users to distinguish scaphocephaly from dolichocephaly.

Strictly speaking, dolichocephaly
and  scaphocephaly  are  descriptors  of  types  of  head  shape
and not of an underlying cause. Dolichocephaly is a term in
wide use to describe any long narrow head, with a cephalic
index usually below 75%. Scaphocephaly
, derived from the
Greek  skaphe  (a  light  boat  or  skiff),  describes  a  specific
variety  of  a  long  narrow  head  that  resembles  an  inverted
boat with a keel. Von Baer
3
was the first to use this term for
sagittal synostosis in 1860 and he wrote “I recommend this
to be named scaphocephaly because it resembles an invert-
ed boat.” Thus, all patients with scaphocephaly suffer from
dolichocephaly
, but the reverse is not true

In summary, we would suggest that an infant with a long,
narrow  head  can  be  described  as  being  dolichocephalic.
This is simply a description of head shape and there is a dif-
ferential diagnosis for the underlying causes. One of these
causes  is  sagittal  synostosis,  and  in  that  scenario  one  ex-
pects to notice the appearance of scaphocephaly, namely an
inverted  keellike  appearance  to  the  skull,  with  the  widest
part of the skull in the temporal rather than parietal area and
with palpable ridging over part or all of the sagittal suture.
It is our opinion that, if one adheres to this, the need for CT

From PMID:    16156241
